release-notes/5.0/api-diff/netcoreapp3.1/.Net/standalone/5.0-standalone_System.Text.Json.md

```diff
@@ -0,0 +1,477 @@
+# System.Text.Json
+
+``` diff
+ {
++    namespace System.Text.Json {
++        public enum JsonCommentHandling : byte {
++            Allow = (byte)2,
++            Disallow = (byte)0,
++            Skip = (byte)1,
++        }
++        public sealed class JsonDocument : IDisposable {
```

Comment:
These types shipped in this OOB during the .NET Core 3.0 wave, not sure why they're showing up as new now...

Reply:
could it be that they were present in different assembly ?

Reply:
The assembly didn't change.  This appears to be listing the entire surface area of System.Text.Json.dll

Reply:
Most of these standalone diffs seem wrong.  They seem to be missing the baseline for 3.x.

Reply:
Taking a look on what went wrong here

Reply:
Its picking up the version of the assembly from the ref pack instead of the package on 3.1 whereas its picking up the assembly from the package on 5.0

Reply:
I removed the packages which are part of the shared framework